README.md

```diff
@@ -6,13 +6,13 @@ Cloudfuse uses file caching to provide the performance of local storage, or you
 Cloudfuse is a fork of [blobfuse2](https://github.com/Azure/azure-storage-fuse), and adds S3 support, a GUI, and Windows support.
 Cloudfuse supports clouds with an S3 or Azure interface.
 
-## Installation
-Download the provided installation packages for your preferred operating system.  -- COMING SOON!
+### Windows installation
 
-### Windows
+ Download and the .exe installer from our latest release [here](https://github.com/Seagate/cloudfuse/releases). Uncheck the "Launch Cloudfuse" upon finishing the installation. Run the GUI separately as admin after the install completes. WinSFP is required for Cloudfuse.
 
- Download and the .exe installer from our latest release [here](https://github.com/Seagate/cloudfuse/releases). Uncheck the "Launch Cloudfuse" upon finishing the installation. Run the GUI separately as admin after the install completes.
+### Linux installation
 
+ Download and the .deb or .rpm installer from our latest release [here](https://github.com/Seagate/cloudfuse/releases). Uncheck the "Launch Cloudfuse" upon finishing the installation. Run the GUI separately as admin after the install completes.

```

Comment:
This needs to be changed, as this is not how the installation works for Linux. It needs to be done on the command line. You can remove the part about unchecking the launch cloudfuse. And instead of running as admin, you would need to run as root.

So you could change it to something like this with these headers under the linux section.

Debian / Ubuntu
sudo dpkg -i ./cloudfuse.deb

CentOS / RHEL
sudo rpm -i ./cloudfuse.rpm

Reply:
David and I talked about this a bit. I think users could install the package from their file explorer.
But I agree that we should also include command-line instructions for installation.
Speaking of which, do we need to add another line for them to install fuse... something like this?:
`sudo apt-get install -f`
I guess the best way to find out is to try it.